src/main/kotlin/com/demonwav/mcdev/i18n/inspections/TranslationInspection.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2017 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.i18n.inspections
+
+import com.intellij.codeInspection.BaseJavaLocalInspectionTool
+import com.intellij.codeInspection.ProblemsHolder
+import com.intellij.psi.JavaElementVisitor
+import com.intellij.psi.PsiClass
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiElementVisitor
+import com.intellij.psi.PsiField
+import com.intellij.psi.PsiFile
+import com.intellij.psi.PsiMethod
+
+abstract class TranslationInspection : BaseJavaLocalInspectionTool() {
+    override fun buildVisitor(holder: ProblemsHolder, isOnTheFly: Boolean): PsiElementVisitor {
+        return object : JavaElementVisitor() {
```

Comment:
To save performance in projects which don't even use MCP, you can just return an empty visitor here, then all checks are simply checked. For Mixin, I do it like this: [MixinInspection](https://github.com/minecraft-dev/MinecraftDev/blob/dev/src/main/kotlin/com/demonwav/mcdev/platform/mixin/inspection/MixinInspection.kt)